dev/com.ibm.ws.threading/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -21,7 +21,7 @@ name=Name
 name.desc=The name of the Liberty kernel default executor.
 
 core.threads=Core threads
-core.threads.desc=Steady-state or core number of threads to associate with the executor. The number of threads associated with the executor will quickly grow to this number. If this value is less than 0, a default value is used. This default value is calculated based on the number of hardware threads on the system.
+core.threads.desc=Baseline or minimum number of threads to associate with the executor. The number of threads associated with the executor will quickly grow to this number. If this value is less than 0, a default value is used. This default value is calculated based on the number of hardware threads on the system. The minimum number of default executor threads Liberty will use is 4.
 
 max.threads=Maximum threads
-max.threads.desc=Maximum number of threads that can be associated with the executor. If greater than 0, this value must be greater than or equal to the value of coreThreads. If the value of maxThreads is less than or equal to 0, the maximum number of threads is unbounded.  Note that the actual number of threads associated with the executor is determined dynamically by the Liberty kernel, so leaving the maximum threads unbounded does not imply that the runtime will actively create large amounts of threads; it simply lets the Liberty kernel decide how many threads to associate with the executor without having a defined upper boundary.
+max.threads.desc=Maximum number of threads that can be associated with the executor. If greater than 0, this value must be greater than or equal to the value of coreThreads (which will be a minimum of 4). If the value of maxThreads is less than or equal to 0, the maximum number of threads is unbounded.  Note that the actual number of threads associated with the executor is determined dynamically by the Liberty kernel, so leaving the maximum threads unbounded does not imply that the runtime will create large amounts of threads; it simply lets the Liberty kernel decide how many threads to associate with the executor without having a defined upper boundary.
```

Comment:
This statement seems overly complex,
"If greater than 0, this value must be greater than or equal to the value of coreThreads (which will be a minimum of 4)."
This could be more directly stated as:
"The value must be greater than or equal to 4 (the minimum for coreThreads) or less than or equal to 0."

Reply:
As with coreThreads, the implementation of maxThreads is a little bit complicated ... if they specify maxThreads less than coreThreads, we reduce coreThreads to be equal to maxThreads ... but coreThreads must be at least 4. The code as per the change in ExecutorServiceImpl is not complicated, but expressing the result clearly and succinctly in English is a bit more challenging. :-)

Reply:
OK, I tried again to make this wording clear and succinct ... let me know what you think. :-)